evm-config.schema.json

```
@@ -20,6 +20,11 @@
         "chrome"
       ]
     },
+    "preserveXcode": {
+      "description": "Whether to delete unused Xcode versions",
+      "type": "number",
+      "default": 5
```

```suggestion
      "type": "integer",
      "default": 5,
      "minimum": 1
```

Leverage validation here to prevent any weirdness.